library/backend.cpp

```diff
@@ -645,9 +646,21 @@ static int dnet_cmd_backend_control_dangerous(struct dnet_net_state *st, struct
 
 	const dnet_backend_info &backend = backends[control->backend_id];
 	if (backend.state == DNET_BACKEND_UNITIALIZED) {
-		dnet_log(node, DNET_LOG_ERROR, "backend_control: there is no such backend: %u, state: %s",
-				control->backend_id, dnet_state_dump_addr(st));
-		return -EINVAL;
+		switch(dnet_backend_command(control->command)) {
+			/* uninitialized backend can be enabled */
+			case DNET_BACKEND_ENABLE:
```

Comment:
On startup, Elliptics marks all backends without config as uninitialized. These changes allow to enable backends which config were added.